### ISSUE TITLE
Getting reponse as "Processing" or "InQueue" for some response

### ISSUE THREAD

**Issue by taneshgupta** [judge0/judge0#324]
When creating batch submission for a program with multiple testcases, for some responses getting status id as 1 or 2 i.e, "InQueue" or "Processing".

**Comment by hermanzdosilovic** [comment]
Do they ever go into "Accepted" or anything else besides those two?

**Comment by taneshgupta** [comment]
Yes they do go into "Accepted" state but after some time. So as a result I have to manually add a setTimeout delay before sending request which is, not effective in my opinion

**Comment by mohitxskull** [comment]
> Yes they do go into "Accepted" state but after some time. So as a result I have to manually add a setTimeout delay before sending request which is, not effective in my opinion

you can save submission details in your database.

and fetch them in interval of 2-5 seconds from your frontend.

add callback_url in your submission so that when your code execution is completed you can update in your database and show it your user.

code execution doesn't happen as soon as you create submission.
